src/ILLink.RoslynAnalyzer/DiagnosticDescriptors.cs

```diff
@@ -59,6 +59,8 @@ public static DiagnosticDescriptor GetDiagnosticDescriptor (DiagnosticId diagnos
 		static string GetDiagnosticCategory (DiagnosticId diagnosticId)
 		{
 			switch ((int) diagnosticId) {
+			case 2117:
```

Comment:
I think we should move this to the 3000+ range.
Maybe something like 3050 or so.

Reply:
Note that below you also have 2118 which is not even covered by this switch.

Reply:
Changed to 3050 and 3051